app/models/application_record.rb

```diff
@@ -17,6 +17,13 @@ def self.arel_find(collection, batch_size = 1000)
     end.inject([], &:+)
   end
 
+  def self.search_for(*args)
+    super
```

Comment:
```
NoMethodError (super: no superclass method `search_for'
```

Reply:
Yeah, wrestling with that :see_no_evil:

Reply:
here is what scoped_search does:
```
register_named_scope! unless klass.respond_to?(:search_for)
```
we need to do other magic...
probably adding a brand new class method.

Reply:
I can redefine `respond_to` to answer correctly :rofl:

Reply:
I am going to figure this out :laughing: no worries